src/main/scala/fundamentals/level03/TryExercises.scala

```diff
@@ -103,16 +93,23 @@ object TryExercises {
     * scala> mkEmployee("Bob,22,abc")
     * = Left(TryError("abc cannot be converted to Boolean"))
     *
-    * Hint: Use csv.split(","), parseIntSafeEither and parseBooleanSafeEither
+    * Hint: Use parseIntSafeEither, parseBooleanSafeEither and for-comprehension
     */
-  def mkEmployee(csv: String): Either[TryError, Employee] = ???
+  def mkEmployee(csv: String): Either[TryError, Employee] =
```

Comment:
Try also has a `toEither` method:

```
sealed abstract class Try[+T] ... {
  def toEither[T]: Either[Throwable, T]
}
```

Reply:
Should we return `Try[Employee]` here? That way we can show the sequencing of `Try` in a `for-comp`